packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -303,7 +303,13 @@ export class SignIn extends BaseResource implements SignInResource {
      * The UI should always prevent from this method being called if WebAuthn is not supported.
      * As a precaution we need to check if WebAuthn is supported.
      */
-    if (!isWebAuthnSupported()) {
+
+    const _isWebAuthnSupported = SignIn.clerk.__unstable__isWebAuthnSupported || isWebAuthnSupported;
```

Comment:
To make it more readable I would prefer that we rename on import and keep the initial value name without the underscore. 
Example:
```
import { isWebAuthnSupported as isWebAuthnSupportedOnWindow } from '@clerk/shared/webauthn';

// and then
const isWebAuthnSupported = SignIn.clerk.__unstable__isWebAuthnSupported || isWebAuthnSupportedOnWindow;
```